### ISSUE TITLE
Finetuning on the Crowd Human dataset to improve people detection

### ISSUE THREAD

**Issue by simpad2409** [Visual-Behavior/detr-tensorflow#31]
Hello everyone and thanks in advance to those who decide to help me. I will be really grateful to him. I am working on my master's thesis and I would like to perform a finetuning on the Crowd Human dataset, in order to improve the detection of people.
But I think I'm doing something wrong ... The model I get is bad! The results on the test set are much worse than the original DETR network.

Could anyone help me?

Thank you so much. I would be immensely grateful.

(I could attach the code, although I followed that guide step by step:
[Link](https://github.com/Visual-Behavior/detr-tensorflow/blob/main/notebooks/DETR%20Tensorflow%20-%20%20Finetuning%20tutorial.ipynb)
modifying only the dataset (train and validation) and the two annotation files.

Please help me. Thanks in advance.

**Comment by ni3s** [comment]
comment here if you still need any help
